api-test/ApiConfig.postman_collection.json

```diff
@@ -1310,6 +1433,11 @@
 							}
 						}
 					],
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {
+							"content-type": true
+						}
+					},
 					"request": {
 						"method": "POST",
 						"header": [
```

Comment:
this call failed when is converted for k6 test because the header Content-Type is lost during conversion